opensafely/check.py

```diff
@@ -41,7 +41,7 @@ class RestrictedDataset:
         cohort_extractor_function_names=[
             "with_an_ons_cis_record",
         ],
-        ehrql_table_names=["ons_cis_raw", "ons_cis"],
+        ehrql_table_names=[],
```

Comment:
Should we remove the entire `RestrictedDataset`?  The function `with_an_ons_cis_record` no longer exists in cohort-extractor; if we're removing the ehql references, we probably should do the same for cohort-extractor

Reply:
We should! I hadn't realized -- or more likely, I had forgotten -- that we removed ONS-CIS from cohort-extractor too.